apps/backoffice-v2/src/lib/blocks/components/NodeBlock/NodeBlock.tsx

```diff
@@ -0,0 +1,4 @@
+import { INodeBlockProps } from '@/lib/blocks/components/NodeBlock/interfaces';
+import { FunctionComponent } from 'react';
+
+export const NodeBlock: FunctionComponent<INodeBlockProps> = ({ value }) => <>{value}</>;
```

Comment:
Let's try and think of a name that better captures what we're trying to achieve `GenericCell` comes to mind. This is a cell that just renders the `value` property or renders any JSX.

Reply:
NodeBlock seems alright for this.